components/utils/PopulationMultiSelect.tsx

```diff
@@ -0,0 +1,184 @@
+import { Group, MultiSelect, Popover, Text, ActionIcon } from '@mantine/core';
+import { InfoCircle } from 'tabler-icons-react';
+import produce from 'immer';
+import { Enums } from 'fqm-execution';
+import { useState } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { measureBundleState } from '../../state/atoms/measureBundle';
+import { patientTestCaseState } from '../../state/atoms/patientTestCase';
+import { selectedPatientState } from '../../state/atoms/selectedPatient';
+
+interface MultiSelectData {
+  value: string;
+  label: string;
+  disabled: boolean;
+}
+
+export default function PopulationMultiSelect() {
+  const measureBundle = useRecoilValue(measureBundleState);
+  const selectedPatient = useRecoilValue(selectedPatientState);
+  const measure = measureBundle.content?.entry?.find(e => e.resource?.resourceType === 'Measure')
+    ?.resource as fhir4.Measure;
+  const [currentPatients, setCurrentPatients] = useRecoilState(patientTestCaseState);
+  const [value, setValue] = useState<string[]>(
+    selectedPatient ? currentPatients[selectedPatient].desiredPopulations || [] : []
+  );
+  const [opened, setOpened] = useState(false);
+
+  /**
+   * Compiles an array of data for the MultiSelect component, using population codes
+   * from the measure resource, across all groups in the measure.
+   *
+   * Excludes measure-population, measure-population-exclusion, and measure-observation
+   * (used for CV measures).
+   * @param {fhir4.Measure} measure - FHIR measure resource
+   * @returns {Array} array of strings representing all unique measure populations
+   */
+  const getMeasurePopulations = (measure: fhir4.Measure): Array<MultiSelectData> => {
```

Comment:
Consistency nitpick haha

```suggestion
  const getMeasurePopulations = (measure: fhir4.Measure): MultiSelectData[] => {
```